actions/release-it/.release-it.js

```diff
@@ -0,0 +1,54 @@
+const version = "${version}";
+const packageName = process.env.npm_package_name;
+const monorepo = process.env.monorepo;
+const branch_name = process.env.branch_name;
+const scope = packageName.split("/")[1];
+
+const releaseBranches = ["main", "master", "release/**"];
+let increment = undefined;
+
+// Check if branch conditions are met
+if (monorepo) {
+  if (branch_name.indexOf("release") > -1 && branch_name.indexOf(scope) < 0) {
+    console.log(`Skipping release of ${scope} since it should not be released from branch ${branch_name}`);
+    process.exit(0);
+  }
+}
+if (branch_name.indexOf("release") > -1) {
+  increment = "patch";
+}
+
+module.exports = {
+  increment,
+  plugins: {
+    "@release-it/conventional-changelog": {
+      path: ".",
+      infile: "CHANGELOG.md",
+      preset: "conventionalcommits",
+      gitRawCommitsOpts: {
+        path: ".",
+      },
+    },
+  },
+  git: {
+    push: true,
+    tagName: `${packageName}-v${version}`,
+    commitsPath: ".",
+    commitMessage: `chore(${scope}): released version v${version} [no ci]`,
+    requireCommits: true,
+    requireCommitsFail: false,
+    requireBranch: releaseBranches,
+  },
+  npm: {
+    publish: false,
+    versionArgs: ["--workspaces false"],
+  },
+  github: {
+    release: true,
+    releaseName: `${packageName}-v${version}`,
+  },
+  hooks: {
+    "before:git:release": ["mvm-update", "git add --all"],
```

Comment:
mvm-update is the trick here. It adds a mvm.lock file in the packages that use internal dependencies to check if they should also be bumped. We should check if we can just use this though.